application/src/ext/java/org/opentripplanner/ext/fares/impl/OrcaFareService.java

```diff
@@ -348,11 +361,22 @@ private Optional<Money> getSeniorFare(
     Leg leg
   ) {
     var route = leg.getRoute();
-    var regularFare = getRegularFare(fareType, rideType, defaultFare, leg);
+    if (route == null) {
```

Comment:
In what cases can the route be null?

Reply:
I don't know, IntelliJ was complaining about a possible NPE so I thought I'd just add this to be safe